### DISCUSSION TITLE
Is there anyway to set up authentication on sqld?

### DISCUSSION THREAD

**Discussion by RenFontes** [tursodatabase/libsql#154] [Q&A]
This project looks so perfect for my needs, just would really like to have authentication when running the db as a server. 🤔

**Comment by EtienneK** [comment]
Have you ever gotten an answer for this? This is the first hit on google for "sqld authentication".

**Comment by cenuij** [comment]
You can add JWT auth ... https://github.com/tursodatabase/libsql/blob/main/docs/USER_GUIDE.md#client-authentication

**Comment by samifouad** [comment]
it's really bad how poorly documented this is

setting up authentication is the first thing we should be learning when working with an http-enabled database
